lib/publiccloud/gce.pm

```diff
@@ -25,9 +26,10 @@ has private_key         => undef;
 has service_acount_name => undef;
 has client_id           => undef;
 
+our $credentials_file = 'google_credentials.json';
```

Comment:
Maybe we should use the mojolicious way of constants: https://mojolicious.org/perldoc/constant

Reply:
ok